axum-extra/Cargo.toml

```diff
@@ -48,8 +48,8 @@ __private_docs = [
 ]
 
 [dependencies]
-axum = { path = "../axum", version = "0.8.2", default-features = false, features = ["original-uri"] }
-axum-core = { path = "../axum-core", version = "0.5.1" }
+axum = { path = "../axum", version = "0.8.3", default-features = false, features = ["original-uri"] }
```

Comment:
`axum-extra` itself is missing a version bump.

Reply:
Yeah, this was intentional as I made this PR before https://github.com/tokio-rs/axum/pull/3293. Now that that's merged, there's no more reason to not release axum-extra.